web/src/engine/websites/MangAs.ts

```diff
@@ -0,0 +1,55 @@
+import { Tags } from '../Tags';
+import icon from './MangAs.webp';
+import { DecoratableMangaScraper, type Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as MangaReader from './decorators/MangaReaderCMS';
+import { RateLimit } from '../taskpool/RateLimit';
+import { Numeric } from '../SettingsManager';
+import { WebsiteResourceKey as R } from '../../i18n/ILocale';
+import type { Priority } from '../taskpool/TaskPool';
+
+const chapterScript = `
+    new Promise((resolve, reject) => {
+        const path = window.location.pathname +'/';
+        resolve(jschaptertemp.map(chapter =>  { 
+            return {
+                id : path + chapter.slug,
+                title : chapter.number + ' : ' + chapter.name.trim() 
+            }}));
+    });
+`;
+
+function MangaExtractor(anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname,
+        title: anchor.querySelector('img').getAttribute('title').trim()
+    };
+}
+
+@Common.MangaCSS(/^{origin}\/manga\/[^/]+$/, MangaReader.queryMangaTitle)
+@Common.MangasMultiPageCSS('/filterList?page={page}&sortBy=name', 'div.media a.thumbnail', 1, 1, 500, MangaExtractor )
+@Common.ChaptersSinglePageJS(chapterScript, 500)
+@Common.PagesSinglePageCSS(MangaReader.queryPages, MangaReader.ChapterPageExtractor)
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('mangas', `MangAs`, 'https://mangas.in', Tags.Language.Spanish, Tags.Media.Manga, Tags.Media.Manhua, Tags.Media.Manhwa, Tags.Source.Aggregator);
+        this.Settings.throttle = new Numeric('throttle.images', R.Plugin_Settings_ThrottlingInteraction, R.Plugin_Settings_ThrottlingInteractionInfo, 4, 1, 60);
```

Comment:
Setting description and RateLimit do not match.
Description says requests/minute but RateLimit is set to requests per 5 seconds window.

Reply:
I see. Are you perhaps suggesting another description?

Reply:
It would be confusing for me as user if every website uses different units for rate limit. Its probably better to just define the RateLimit in requests per minute instead of requests per 5 seconds.
`new RateLimit(..., 60);`